pkg/client/unversioned/daemon_sets_test.go

```diff
@@ -31,12 +36,12 @@ func getDSResourceName() string {
 
 func TestListDaemonSets(t *testing.T) {
 	ns := api.NamespaceAll
-	c := &testClient{
-		Request: testRequest{
+	c := &simple_testclient.TestClient{
```

Comment:
If we move TestClient into test pkg, the struct probably should be renamed to Client. Then we do not need to repeat the pkg name in the struct.